src/components/landing-page/index.tsx

```diff
@@ -0,0 +1,65 @@
+import { LANDING_PAGE_FEATURES } from "@/lib/constants/misc";
+import { Button } from "../ui/button";
+import Image from "next/image";
+import { grayBlurDataValue } from "@/lib/utils";
+
+const LandingPage = () => {
+  return (
+    <>
+      <div className="bg-[url('/images/app_landing.jpg')] bg-center md:h-[400px] bg-cover rounded-[12px] p-[40px]">
+        <div className="h-full flex flex-col justify-end">
+          <div className="space-y-[20px]">
+            <h1 className="text-[36px] text-white font-[700] md:text-[48px]">
+              SpendWise helps you spend smarter and save more.
+            </h1>
+            <p className="text-white text-[18px] font-[200]">
+              Understanding your spending habits, track recurring expenses, and
+              set budgets to meet your financial goals. Whether you're saving
+              for a big purchase or just want to be more mindful of your
+              spending, SpendWise has you covered.
+            </p>
+            <div className="flex flex-col space-y-[20px] xs:flex-row xs:space-y-0 xs:space-x-[20px] !mt-[40px]">
+              <Button size="lg">Sign up</Button>
```

Comment:
is it necessary to use important here?

Reply:
@DhanBirSunwar  yup, parent has `space-y-` style so need to override that.